airflow/config_templates/default_airflow.cfg

```diff
@@ -324,6 +324,12 @@ error_logfile = -
 # Expose the configuration file in the web server
 expose_config = False
 
+# Expose hostname in the web server
+expose_hostname = True
```

Comment:
Can you add some documentation about this option?   If there is no information about this feature in the documentation, very few people will be able to use it.

Reply:
@mik-laj - > where do you think the documentation should be added ? Maybe you can provide some pointers? Most of the config options are documented in comments of the default_airlfow.cfg. Any other specific place you think this documentation should be added (and where all the previous options are documented?)

Reply:
I answered most of your questions in a different thread.:
https://github.com/apache/airflow/pull/6923#discussion_r362110552

I think we can add  new section in `security.rst`file. This file is long and suitable for refactoring, but I think we should do it in a separate PR. Please note that refactoring documentation, which is even a collection of random guides, is simpler than writing all documentation from scratch.

Reply:
A similar situation took place with the guides for operators.  Initially, we only had one file that contained a description of all operators. However, at some point, we refactored the documentation to create a new quality.

Reply:
@tooptoop4 -> would you be so kind to add this in the doc section as explained by @mik-laj ?